server.js

```diff
@@ -461,6 +480,24 @@ app.post('/fetchqlist', function(req, res) {
         ans = true;
     }
 
+    var request = {};
+    request.user = req.session.user;
+    request.questionsStatus = req.body.type;
+    
+    users.getQuestionsList(request, function(err, questionsList){
+        if (err) {
+            return res.status(500).send('Could not fetch questions list');
+        }
+
+        req.session.adminQuestionList = null;
+        var html = questionList({
+            questions: questionsList
+        });
+
+        return res.status(200).send(html);
+    });
+
+    /*
```

Comment:
Should have explain of why you commented out this code. Do you plan on using it later? Has it been replaced and will be deleted after?

Reply:
oooh my bad. I am trying to replace it. To make sorting easier for future addition